docs/D-Modularity/output-functions.md

```diff
@@ -1,6 +1,10 @@
 ---
 sidebar_position: 5
+title: Output Functions
+id: output-functions
+slug: /modularity/output-functions
```

Comment:
I think you should add description below the `slug` as well to let others know about the content in this page